File: LV1/1.8/statistics/statistics.py
ts = 0
while True:
    ts += 1
    try:
        line = [int(i) for i in input().split(' ')][1:]
        a = min(line)
        b = max(line)
        print(f'Case {ts}: {a} {b} {b-a}')
    except EOFError:
        break

File: LV1/1.8/statistics/statistics.rs
use std::io::{self, BufRead};

fn main() {
    let stdin = io::stdin();
    let mut ts = 1;

    for line in stdin.lock().lines().map(|l| l.unwrap()) {
        let nums: Vec<i32> = line
            .split_whitespace()
            .skip(1)
            .map(|n| n.parse().unwrap())
            .collect();
        let a = nums.iter().min().unwrap();
        let b = nums.iter().max().unwrap();

        println!("Case {}: {} {} {}", ts, a, b, b - a);
        ts += 1;
    }
}
